cmd/kubeadm/app/util/system/selinux.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package system
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/golang/glog"
+	selinux "k8s.io/kubernetes/pkg/util/selinux"
+)
+
+type SELinuxContext struct {
+	User  string
+	Role  string
+	Type  string
+	Level string
+}
+
+// Create a new SELinux context by default allowing files to be shared
+// with containers
+func NewSELinuxContext() *SELinuxContext {
+	return &SELinuxContext{
+		"unconfined_u", "object_r", "container_file_t", "s0"}
+}
+
+// Converts the context into an SELinux label
+func (c SELinuxContext) ToString() string {
+	arr := []string{c.User, c.Role, c.Type, c.Level}
+	return strings.Join(arr, ":")
```

Comment:
It will be way better in my opinion to avoid creating an array and slice here. Just use `fmt.Sprintf`.

Reply:
I started off with sprintf, but was wondering if I needed to make it configurable based on giving different levels to different processes after getting some feedback. I can retreat back now though.

Reply:
I'd suggest to start by keeping this simple and enhance on it if it's needed :)